File: official/docs/golang/current/shipping-insurance/insure.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func insure() {
	client := easypost.New("EASYPOST_API_KEY")

	shipment, _ := client.InsureShipment("shp_...", "100.00")

	fmt.Println(shipment)
}

File: official/docs/csharp/v5/shipping-insurance/insure.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            EasyPost.Models.API.Shipment shipment = await client.Shipment.Retrieve("shp_...");

            shipment = await client.Shipment.Insure(shipment.Id, 200);

            Console.WriteLine(JsonConvert.SerializeObject(shipment, Formatting.Indented));
        }
    }
}
